kiezcoding.html

```diff
@@ -84,7 +84,7 @@
 
             <h2>How much is it?</h2>
 
-            <p>Regular co-working day passes at co.up are 15 euro per day. For our special event, there will be a need-based sliding scale from 0-15 euro. If you are able to, <a href="https://members.co-up.de/membership_signup/new?plan_id=261dfabc9ec1ebf95aefa14246b80c4c">register for a regular priced co.up day pass</a>. If you are not able to pay the full price for any reason, <a href="mailto:WHICH_EMAIL_ADDRESS">email us to make the arrangements</a>.</p>
+            <p>Regular co-working day passes at co.up are 15 euro per day. For our special event, there will be a need-based sliding scale from 0-15 euro. If you are able to, <a href="https://members.co-up.de/membership_signup/new?plan_id=261dfabc9ec1ebf95aefa14246b80c4c">register for a regular priced co.up day pass</a>. If you are not able to pay the full price for any reason, please <a href="mailto:info@berlinjs.org">email us to make the arrangements</a>.</p>
```

Comment:
I'm out for lunch, but everything looks good to me. Ship it :)

> On 01.04.2016, at 12:32, Michele <notifications@github.com> wrote:
> 
> In kiezcoding.html:
> 
> > @@ -84,7 +84,7 @@
> >  
> >              <h2>How much is it?</h2>
> >  
> > -            <p>Regular co-working day passes at co.up are 15 euro per day. For our special event, there will be a need-based sliding scale from 0-15 euro. If you are able to, <a href="https://members.co-up.de/membership_signup/new?plan_id=261dfabc9ec1ebf95aefa14246b80c4c">register for a regular priced co.up day pass</a>. If you are not able to pay the full price for any reason, <a href="mailto:WHICH_EMAIL_ADDRESS">email us to make the arrangements</a>.</p>
> > +            <p>Regular co-working day passes at co.up are 15 euro per day. For our special event, there will be a need-based sliding scale from 0-15 euro. If you are able to, <a href="https://members.co-up.de/membership_signup/new?plan_id=261dfabc9ec1ebf95aefa14246b80c4c">register for a regular priced co.up day pass</a>. If you are not able to pay the full price for any reason, please <a href="mailto:info@berlinjs.org">email us to make the arrangements</a>.</p>
> I do think it was okay before, because some of the responsibility does lie on the person in question. i just threw it in as a quick change without thinking too deeply about it.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>